### PR TITLE
Selectable table bug fix

### PR DESCRIPTION
**Pull request by GowthamShanmugam**

### CODE REVIEW

File: packages/shared/src/table/selectable-table.tsx [hunk 1]
```diff
@@ -50,6 +63,8 @@ export const SelectableTable: SelectableTableProps = <
     RowComponent,
     extraProps,
     isSelectableHidden,
+    loaded,
+    loadError,
   } = props;
   const { t } = useCustomTranslation();
   const {
```
Comment (SanjalKatiyar) [inline packages/shared/src/table/selectable-table.tsx:70]: need to maintain same reference for `new Set(selectableRows?.map(getUID)),` (https://github.com/red-hat-storage/odf-console/pull/926/files#diff-a776ebfbd967acff8db4670cb1c227fa5d8dac4d7739f9dbb1cab12da92eb5d1R81) across re-render, which is an argument for `useSelectList` below...

Comment (SanjalKatiyar) [reply packages/shared/src/table/selectable-table.tsx:70]: same is true for `selectableRows` as well, need to maintain the reference...

Comment (SanjalKatiyar) [reply packages/shared/src/table/selectable-table.tsx:70]: cc @GowthamShanmugam

Comment (GowthamShanmugam) [reply packages/shared/src/table/selectable-table.tsx:70]: i forgot to push this change :)

Comment (GowthamShanmugam) [reply packages/shared/src/table/selectable-table.tsx:70]: changes done

**Pull request merged**